src/poprox_recommender/model/nrms/user_encoder.py

```diff
@@ -7,16 +7,15 @@
 
 
 class UserEncoder(torch.nn.Module):
-    def __init__(self, config):
+    def __init__(self, hidden_size, num_attention_heads):
```

Comment:
Need to save these parameters (or manually implement to/from config)

Reply:
Maybe I'm missing something about how the component serialization works, but...these are Torch model classes?

Reply:
Since the models will get loaded from disk again when the components are reconstructed from their configs, I don't get why it would matter here

Reply:
You're right, false positive in my detection of configurable classes.